app/ptxboa_functions.py

```diff
@@ -138,6 +138,10 @@ def calculate_results_list(
             else:
                 settings.update({"secproc_co2": None})
 
+        # for all regions but the selected one, use Wind-PV-hybrid RE source:
+        if settings["region"] != st.session_state["region"]:
+            settings["res_gen"] = "Wind-PV-Hybrid"
+
```

Comment:
@markushal I think this can mess things up if the region in session state is not at the first position in the parameter list. Would need to change to:
```python
        # for all regions but the selected one, use Wind-PV-hybrid RE source:
        if settings["region"] == st.session_state["region"]:
            settings["res_gen"] = st.session_state["res_gen"]
        else:
            settings["res_gen"] = "Wind-PV-Hybrid"
```

Reply:
Good point! Thanks!